src-java/testing/functional-tests/src/main/groovy/org/openkilda/functionaltests/helpers/PortAntiflapHelper.groovy

```diff
@@ -33,9 +33,7 @@ class PortAntiflapHelper {
         def swPort = new Tuple2(swId, portNo)
         def lastEvent = history.get(swPort)
         if (lastEvent) {
-            Wrappers.wait(antiflapCooldown + WAIT_OFFSET) {
-                assertPortIsStable(swId, portNo, lastEvent)
-            }
+            assertPortIsStable(swId, portNo, lastEvent)
```

Comment:
Since wait is now inside the method, method name/doc should be updated

Reply:
renamed to `waitPortIsStable`